Dockerfile

```diff
@@ -11,11 +11,11 @@
 #   - https://pkgs.org/ - resource for finding needed packages
 #   - Ex: hexpm/elixir:1.15.4-erlang-26.0.2-debian-bullseye-20230612-slim
 #
-ARG ELIXIR_VERSION=1.18.3
-ARG OTP_VERSION=27.3.2
-ARG ALPINE_VERSION=3.21.3
+ARG ELIXIR_VERSION=1.19.3
+ARG OTP_VERSION=28
+ARG ALPINE_VERSION=3.22.2
 
-ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-alpine-${ALPINE_VERSION}"
```

Comment:
as of stated [above](https://github.com/zebbra/data_aggregator/pull/996/files#r2559955263) the hexpm image for elixir 1.19 and otp 28 isnt availabe (yet?)